DataFormats/ForwardDetId/interface/FastTimeDetId.h

```diff
@@ -14,7 +14,7 @@ class FastTimeDetId : public DetId {
   static const int kFastTimeCellPhiMask     = 0x3FF;
   static const int kFastTimeZsideMask       = 0x100000;
```

Comment:
@davidlange6 This is OK for now, the bit offsets all make sense. Please go ahead and merge, I'll fix this style issue in another PR.